playbook.md

```diff
@@ -971,87 +971,70 @@ to build an appropriate hosting platform for their service. This is fair
 enough, because they shouldn't have to. We built GovPress to meet this need,
 and it is now used by most of our clients.
 
+We offer [managed hosting and support](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/930612236449495) for WordPress websites, as well as [blogging](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/355674790119695) and [campaigns](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/691308552308120) platforms, through the G-Cloud framework, configured to our client's branding requirements, all on our GovPress hosting platform.
+
 You can read more about GovPress at
 [www.govpress.com](https://www.govpress.com).
 
-We're in the process of building a new container-based hosting platform which
-will be able to support services that aren't built in WordPress. Watch this
-space.
+### Tailored support
+
+We also offer support for services that aren't built in WordPress. We are able to host and support services with our [tailored support plan for cloud applications](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/393702576050692), even if your service isn't hosted with us.
 
 ### Support helpdesk
 
 The dxw Technical Operations team build, manage and run GovPress, as well as
-provide first-line support.
+providing first-line support for our clients on tailored support plans.
```

Comment:
The only comment I still have: to me it read more fluently with "provide", as in "The [...] team build, manage and run [...], as well as provide". "The team" is the subject, and "build, manage, run, provide" are all actions done by the subject. But I'm not a native speaker, so maybe this is one of those cases where what "sounds" right isn't right grammatically. 🤷‍♀

Reply:
Hmm, that still doesn't sound quite right. Let me revert this change to the original sentence, without mentioning clients on tailored support plans.